Spotify.js

```diff
@@ -188,18 +189,18 @@ class Spotify {
         var req = () => {
             request(authOptions, (error, response, body) => {
                 if (error) {
-                    console.log(`[SPOTIFY] API Request fail on :`, api)
-                    console.log(error, body)
+                    //console.log(`[SPOTIFY] API Request fail on :`, api)
+                    //console.log(error, body)
                 } else {
                     if (api !== "/v1/me/player" && type !== "GET") {
                         //console.log(`[SPOTIFY] API Requested:`, api)
                     }
                 }
                 if (cb) {
-                    if (response.statusCode) {
+                    if (response && response.statusCode) {
```

Comment:
I'm pretty sure this will solve for 

https://github.com/bugsounet/MMM-Spotify/issues/50

Reply:
i correct some bugs in core (with free spotify account)

I will make an feature for debug in Spotify.js

> TypeError: Cannot read property 'statusCode' of undefined
generaly this error is on connexion lost.
I will see